src/Microsoft.Repl/Microsoft.Repl.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
nit : `TargetFramework` instead of `TargetFrameworks`

Reply:
Whoops. Good catch. Changed.